### ISSUE TITLE
Local tests

### ISSUE THREAD

**Issue by koresar** [ericelliott/stampit#61]
There are a couple options:

1. Port the tests to [tape](https://github.com/substack/tape)
2. Keep the QUnit suite and use [phantomjs](http://phantomjs.org/) to get the test results.

Option 1 is probably best, but we'll need to bundle the test scripts for browsers so we can still run them on Sauce Labs, too.

Check out [tape-run](https://github.com/juliangruber/tape-run)

**Comment by ericelliott** [comment]
I vote 1.

**Comment by JosephClay** [comment]
I've ported the QUnit tests to tape.

Things of note: 
- I had to install lodash as a dependency in order to build (#51)
- tape-run requires phantomjs to be installed globally
- browserify has a conflict with grunt-browserify, so I was unable to install it in the package.json to follow [tape-run's convention](https://github.com/juliangruber/tape-run#usage). There is, however, a tape script that can be run if you have browserify installed globally: `npm run tape`

I'm not familiar enough with Sauce Labs to know how to integrate it with tape-run.
